README.md

```diff
@@ -6,6 +6,27 @@ This repository stores user-defined annotations against glean metrics: both thos
 
 The published set of annotations is available at: https://mozilla.github.io/glean-annotations/api.json
 
+[android-components]: https://github.com/mozilla-mobile/android-components
+
 ## Local development
 
-[android-components]: https://github.com/mozilla-mobile/android-components
+In general, the idea behind this repository is that most people will edit annotations from the Glean Dictionary.
+However, if you want to do local development the recommended setup process is to create a virtual environment
+and install the dependencies inside it. This requires a recent version of python (3.7+) as well as
+[virtualenv]:
+
+```bash
+virtualenv -p python3 venv
+venv/bin/activate
+pip install -r requirements.txt
+```
+
+After creating activating a virtual environment and installing the dependencies, run:
+
+```bash
+scripts/create-api.py > api.json
```

Comment:
```suggestion
./scripts/create-api.py > api.json
```

The leading `./` is necessary in bash. Leaving it off makes this fairly zsh-specific.

Reply:
Hmm, are you sure? The original seems to WFM from bash.

Reply:
Do you have `.` as an entry on `$PATH`? 

See https://superuser.com/questions/944973/any-way-to-avoid-dot-slash-when-running-executable-scripts-in-bash
 and https://stackoverflow.com/questions/6331075/why-do-you-need-dot-slash-before-executable-or-script-name-to-run-it-in-bas

Reply:
Nope! I wonder why it works, then. I'll add in your suggestion nonetheless, since it sounds like it's maximally compatible in all cases.